src/Core/Util/InternationalizedDomainNameConverter.php

```diff
@@ -44,6 +44,15 @@ public function emailToUtf8(string $email): string
             return $email;
         }
 
-        return $parts[0] . '@' . idn_to_utf8($parts[1], 0, INTL_IDNA_VARIANT_UTS46);
+
```

Comment:
```suggestion
```
PHP CS Fixer wants to remove this line, else it complains 😄 

https://github.com/PrestaShop/PrestaShop/actions/runs/3088783456/jobs/4995760254